Plugins/Bars.lua

```diff
@@ -1165,22 +1189,28 @@ do
 		if anchor == normalAnchor then up = db.growup else up = db.emphasizeGrowup end
 		for i = 1, #tmp do
 			local bar = tmp[i]
-			local spacing = currentBarStyler.GetSpacing(bar) or db.spacing
-			bar:ClearAllPoints()
-			if up or (db.emphasizeGrowup and bar:Get("bigwigs:emphasized")) then
-				if lastBar then -- Growing from a bar
-					bar:SetPoint("BOTTOMLEFT", lastBar, "TOPLEFT", 0, spacing)
-				else -- Growing from the anchor
-					bar:SetPoint("BOTTOMLEFT", anchor, "BOTTOMLEFT")
-				end
-				lastBar = bar
+			local barLimit = bar:Get("bigwigs:emphasized") and db.visibleBarLimitEmph or db.visibleBarLimit
+			if  i>barLimit then
+				bar:Hide()
```

Comment:
Double space next to `if`

You cannot use `:Hide()`
If you enable limited bars for the emphasized anchor and create a whole bunch of test bars you will start to notice the little glitches that appear on the bars that are being shown, such as momentarily showing as full, or bars being out of order. This is because when you hide the bar it kills the update timer which is setting the .remaining value.

You will need to use alpha values.